projects/xy-finance/index.js

```diff
@@ -0,0 +1,48 @@
+const utils = require('../helper/utils')
+
+const chains = {
+  1: 'ethereum',
+  10: 'optimism',
+  25: 'cronos',
+  56: 'bsc',
+  108: 'thundercore',
+  137: 'polygon',
+  250: 'fantom',
+  321: 'kcc',
+  592: 'astar',
+  1285: 'moonriver',
+  42161: 'arbitrum',
+  43114: 'avax',
```

Comment:
這個名稱如果沒有一定要是 `'avax'` 的話，我覺得可以直接改成 `avalanche`，這樣下面 `chainTvls` 就不用特別為 avalanche 判斷了。
```suggestion
  43114: 'avalanche',
```

Reply:
Make sense!